src/test/scala/no/ndla/draftapi/controller/NdlaControllerTest.scala

```diff
@@ -0,0 +1,28 @@
+/*
+ * Part of NDLA draft_api.
+ * Copyright (C) 2021 NDLA
+ *
+ * See LICENSE
+ */
+
+package no.ndla.draftapi.controller
+
+import no.ndla.draftapi.{TestData, TestEnvironment, UnitSuite}
+import no.ndla.draftapi.model.api.UpdatedArticle
+import org.scalatra.swagger.SwaggerEngine
+
+class NdlaControllerTest extends UnitSuite with TestEnvironment {
+
+  val ndlaController = new NdlaController {
+    override protected implicit def swagger: SwaggerEngine = ???
+
+    override protected def applicationDescription: String = ???
+  }
+
+  test("That extraction of request body parses relatedContent correctly") {
+    val updatedArticle = ndlaController.extract[UpdatedArticle]("""{"revision":2,"relatedContent":[1]}""").get
+
+    updatedArticle should be(TestData.blankUpdatedArticle.copy(revision = 2, relatedContent = Some(Seq(Right(1L)))))
```

Comment:
Denne testen tester jo eeegentlig ikke feilen (selvom jeg syns det er megateit at json4s/scala ikke funker sånn)

Kanskje legge til dette for å teste at vi _faktisk_ får en long?
```suggestion
    updatedArticle should be(TestData.blankUpdatedArticle.copy(revision = 2, relatedContent = Some(Seq(Right(1L)))))
        val Right(x) = updatedArticle.relatedContent.get.head
    x.toLong should be(1L)

```

Reply:
Med disse endringene feiler faktisk testen uten DefaultFormats.withLong. Jeg oppdaterer testen i master med disse linjene.